src/components/ATATSummaryCard.vue

```diff
@@ -11,118 +11,125 @@
+              <div
+                class="type body"
+                v-if="card.type && card.type == 'TASK ORDER'"
+              >
+                {{ card.type }}
+              </div>
+              <div class="d-flex align-start">
+                <v-btn
+                  :id="'header_link_' + index"
+                  :ripple="false"
+                  @click="leftButtonClicked(card)"
+                  small
+                  class="h3 link-button no-focus-shift pa-0"
+                  :class="{ 'no-border': card.type === 'APPLICATION' }"
+                >
+                  <div class="font-weight-bold">
+                    <div>{{ card.title }}</div>
+                  </div>
+                </v-btn>
+                <v-btn
+                  small
+                  :ripple="false"
+                  @click="leftButtonClicked(card)"
+                  class="
+                    no-focus-shift
+                    pa-0
+                    no-text-decoration no-border
+                    link-button
+                    mx-n2
+                  "
+                  ><v-icon v-if="card.showChevronRight" x-large>
+                    chevron_right
+                  </v-icon>
+                </v-btn>
+              </div>
+            </div>
+            <div>
+              <v-icon color="success"> check_circle </v-icon>
+            </div>
+          </v-card-title>
+          <v-card-subtitle
+            class="body-lg black--text ml-2"
+            v-if="card.description"
+          >
+            {{ card.description }}
+          </v-card-subtitle>
+
+          <v-card-text
+            class="mx-0 px-6"
+            style="min-height: 125px; max-height: 125px"
+          >
+            <div
+              class="
+                body-lg
+                d-flex
+                justify-space-between
+                black--text
+                grouped-items
+              "
+              v-for="(item, idx) in card.items"
+              :key="idx"
             >
-              <div class="font-weight-bold">
-                <div>{{ card.title }}</div>
+              <div class="ml-4">{{ item.title }}</div>
+              <div v-if="item.value">
+                <div>{{ item.prefix }}&nbsp;{{ item.value }}</div>
               </div>
-            </v-btn>
+            </div>
+          </v-card-text>
+          <v-divider></v-divider>
+          <v-card-actions class="d-flex justify-space-between pa-0">
             <v-btn
+              class="
+                link-button
+                no-focus-shift
+                link-body-md
+                text-uppercase
+                my-4
+                mx-6
+                px-2
+                py-0
+              "
               small
-              :ripple="false"
               @click="leftButtonClicked(card)"
+              :id="card.leftButtonText + '_' + index"
+              :ripple="false"
+              >{{ card.leftButtonText }}</v-btn
+            >
```

Comment:
closing `>` on separate line?

Reply:
This is our linter linting a little too much. We need to adjust the rules.